index.js

```diff
@@ -1,17 +1,55 @@
 class SortedList {
-  constructor() {}
+    constructor(items) {
+        this.items = [];
+        this.length = this.items.length;
+    }
 
-  add(item) {}
+    add(item) {
+        this.items.push(item);
+        this.items.sort((a, b) => a - b);
+        this.length = this.items.length;
+    }
 
-  get(pos) {}
+    get(pos) {
+        if (this.items[pos] != undefined) {
+            return this.items[pos];
+        } else {
+            throw new Error('OutOfBounds');
+        }
+    }
 
-  max() {}
+    max() {
+        if (this.length != 0) {
+            return Math.max(...this.items);
+        } else {
+            throw new Error('EmptySortedList');
+        }
+    }
 
-  min() {}
+    min() {
+        if (this.length != 0) {
+            return Math.min(...this.items);
+        } else {
+            throw new Error('EmptySortedList');
+        }
 
-  sum() {}
+    }
 
-  avg() {}
+    sum() {
+        if (this.length != 0) {
+            return this.items.reduce((accu, curr) => accu + curr);
+        } else {
+            return 0;
+        }
+    }
+
+    avg() {
+        if (this.length != 0) {
+            return this.items.reduce((accu, curr) => accu + curr) / this.length;
```

Comment:
Well, you just did that on the function above, you just had to divide sum() by the length